asyncio_run_in_process/exceptions.py

```diff
@@ -12,3 +12,11 @@ class InvalidState(BaseRunInProcessException):
 
 class ChildCancelled(BaseRunInProcessException):
     pass
+
+
+class UnpickleableValue(BaseRunInProcessException):
+    pass
+
+
+class InvalidDataFromChild(BaseRunInProcessException):
+    pass
```

Comment:
consider adding a docstring to document what this exception is used for

Reply:
:+1: